deployment/ci-cd-build-server.md

```diff
@@ -0,0 +1,106 @@
+---
+title: CI, CD, Build Server
+page_title: CI, CD, Build Server
+description: CI, CD, Build Server setup involving the UI for Blazor suite.
+slug: deployment-ci-cd-build-pc
+tags: deploy,deployment,ci,cd,build,pc,machine
+published: True
+position: 5
+---
+
+# CI, CD, Build Server Setup
+
+This article explains some concepts and how to troubleshoot the most common errors related to setting up the Telerik NuGet packages for automated builds, CI and CD.
+
+Sections in this article:
+
+
+* [Basics](#basics)
+* [Azure DevOps Pipelines](#azure-devops-pipelines)
+* [GitHub Secrets](#github-secrets)
+
+## Basics
+
+Often enough, you would want to set up Continuous Integration and/or Continuous Delivery (CI/CD) pipelines or builds for your project that uses the Telerik components. This is a valid scenario and the "one license per developer" license does not prevent you from doing so. The Telerik components are commercial software and as such can only be distributed through channels that are private and/or behind authentication.
+
+There are a couple of common ways people implement CI/CD automated builds.
+
+* You can put your own credentials (or the credentials of the license holder, depending on how your licenses are set up) in the `nuget.config` of the build machine/pipeline. In many cases, when doing so, they will even be encrypted when you add the Telerik feed source through the CLI. Alternatively, you can copy an encrypted version from your own local config if you have one and if plain text is an issue. See more on setting up the [Telerik Private NuGet feed]({%slug installation/nuget%}).
+
+* Creating a local folder (for example, on a shared network drive or other suitable location accessible only by your builds and team) that holds the `.nupkg` files we provide (you can download them from your telerik.com account, or from your local installation - both [automated]({%slug installation/msi%}) and from the [zip archive]({%slug installation/zip%})).
+
+You must protect your credentials and/or the Telerik packages and ensure they are used only by you and not by other developers, according to the [license-per-developer policy](https://www.telerik.com/purchase/license-agreement/blazor-ui). They can by such colleagues (like other developers, QAs, designers, front-end devs, DBAs and so on) for building and running a solution, provided they do not use the Telerik components to create functionality. Of course, you must ensure that such credentials or package sources are not available to the general public (for example, in public repositories). 
```

Comment:
I recommend reworking the 2nd sentence to

"However, as long as your credentials are obfuscated/masked, they can be used by your colleagues (e.g. developers, QAs, designers, front-end devs, DBAs, etc.) for building and running a solution, provided they do not use the Telerik components to create functionality."

I recommend reworking the of last sentence to "Most importantly, you must ensure.."  (instead of using "Of course, you must")

> Using 'of course' carries an implied "Duh", which will make a reader insulted if they never considered the problem before :)

Reply:
Duh :)